docs/PARTITION_TABLE_EDITOR.md

```diff
@@ -0,0 +1,27 @@
+# Partition Table Editor UI for ESP-IDF
+
+Our VS Code Extension comes with UI for editing your [partition table](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/partition-tables.html) and flash it to your chip, instead of dealing with csv files we present you with comfortable UI where you can edit the existing partition table or create a new one of your choosing.
+
+## Prerequisites
+
+- ESP-IDF `>=v4.x`
+- IDF VSCode extension version `>=0.5.0`
+- We assume that you have built your project already and ready to edit the generated partition table directly from the partition binary file.
```

Comment:
Same question — why from the partition binary, and not the CSV?

Reply:
Yes update documentation is also pending for todo in description